llvm/lib/Support/Unix/Memory.inc

```diff
@@ -67,6 +72,24 @@ static int getPosixProtectionFlags(unsigned Flags) {
   return PROT_NONE;
 }
 
+#if defined(__linux__)
+#ifndef MFD_CLOEXEC
+#define MFD_CLOEXEC 0x0001U
+#endif
+#ifndef MFD_EXEC
+#define MFD_EXEC 0x0010U
+#endif
+
+static inline int memfd_create(const char *name, int flags) {
```

Comment:
not super relevant as it's static, but perhaps mimic the libc & syscall prototype, where `flags` is `unsigned int` ?

Reply:
Will do, an oversight of mine!

Reply:
Heh, fixing the `flags` argument type made the function clash with libc's definition of `memfd_create()` so I had to rearrange the code to only define the stub if needed.